From 058ff49a5b3ea5ac5a019c3fb15d7a69e1060450 Mon Sep 17 00:00:00 2001
From: Baji Patthan <bpatth@codeaurora.org>
Date: Tue, 5 May 2015 19:44:28 -0700
Subject: [PATCH 21/22] ARM:dts:msm: Add support of the quaternary I2S

Enable the quaternary I2S configuration which secondary I2S path
on the SBC8016 platform.

Change-Id: I063c960b4dca14dff35267bdb3065cb2d49b1ca9
Signed-off-by: Baji Patthan <bpatth@codeaurora.org>
---
 arch/arm/boot/dts/qcom/apq8016-sbc.dtsi | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/apq8016-sbc.dtsi b/arch/arm/boot/dts/qcom/apq8016-sbc.dtsi
index 4432da0..4c2ad52 100644
--- a/arch/arm/boot/dts/qcom/apq8016-sbc.dtsi
+++ b/arch/arm/boot/dts/qcom/apq8016-sbc.dtsi
@@ -156,7 +156,7 @@
 		qcom,model = "msm8x16-snd-card-sbc";
 		qcom,msm-snd-card-id = <0>;
 		qcom,msm-codec-type = "internal";
-		qcom,msm-ext-pa = "primary";
+		qcom,msm-ext-pa = "quaternary";
 		qcom,msm-mclk-freq = <9600000>;
 		qcom,msm-mbhc-hphl-swh = <0>;
 		qcom,msm-mbhc-gnd-swh = <0>;
@@ -181,7 +181,9 @@
 				"cdc_lines_sec_ext_act",
 				"cdc_lines_sec_ext_sus",
 				"cross_conn_det_act",
-				"cross_conn_det_sus";
+				"cross_conn_det_sus",
+				"cdc_lines_quat_ext_act",
+				"cdc_lines_quat_ext_sus";
 		pinctrl-0 = <&cdc_pdm_lines_act>;
 		pinctrl-1 = <&cdc_pdm_lines_sus>;
 		pinctrl-2 = <&cdc_pdm_lines_act &cdc_ext_pa_act
@@ -190,6 +192,8 @@
 						&cdc_ext_pa_ws_sus>;
 		pinctrl-4 = <&cross_conn_det_act>;
 		pinctrl-5 = <&cross_conn_det_sus>;
+		pinctrl-6 = <&ext_cdc_tlmm_lines_act>;
+		pinctrl-7 = <&ext_cdc_tlmm_lines_sus>;
 
 		asoc-platform = <&pcm0>, <&pcm1>, <&voip>, <&voice>,
 				<&loopback>, <&compress>, <&hostless>,
-- 
1.8.2.1

